### ISSUE TITLE
Questions about training code

### ISSUE THREAD

**Issue by Krasner** [PaulCouairon/JAFAR#9]
I have a few questions about the code that don't match the paper:

1. Why is the image downscaled by 2 before feeding to the image encoder
```
image_batch = F.interpolate(image_batch, scale_factor=0.5, mode="bilinear")
jafar_hr_feats = run_model(jafar, [image_batch, lr_feats, (h, w)])
```
Section 3.1 says:
```
The image I is first projected into a higher-dimensional space and processed by a lightweight encoder Eθ 
to obtain an intermediate representation IE = Eθ(I) ∈ Rd×H×W ,further enriched with RoPE positional embeddings.
```
Why is the interpolation downscaling step needed? Why not use the original `image_batch`?

2. There is also a regularization loss against a blurry image:
```
noised_batch = add_blur_noise(image_batch)
pred_uhr_feats = run_model(jafar, [noised_batch, hr_feats, (224, 224)])
with torch.no_grad():
    target_uhr_feats = run_model(jafar, [image_batch, hr_feats, (224, 224)])
```
I can't find the section in the paper that discusses why this is needed. Also why is (224,224) target size used and not the HR image size of (448,448)?

Thanks!
